### DISCUSSION TITLE
Difference between self-contained and embed-resources

### DISCUSSION THREAD

**Discussion by Ifitrains** [quarto-dev/quarto-cli#3021] [Q&A] [answered]
The quarto [doc](https://quarto.org/docs/output-formats/html-basics.html#self-contained) uses embed-resources, in quarto-tip-a-day-[blog](https://mine-cetinkaya-rundel.github.io/quarto-tip-a-day/posts/09-self-contained/) self-contained is used to produce stand-alone HTML files. Are they the same or are there fundamental differences? Thank you!

**Comment by Ifitrains** [comment]
Nevermind, https://github.com/quarto-dev/quarto-web/issues/422

**Reply by mcanouil** [reply, marked as answer]
This issue has just been closed, leaving the deprecated `self-contained` yaml option only in the reference pages.

**Comment by mcanouil** [comment]
In addition to the issue about updating the documentation, in the reference page you get more details about the two yaml options (see <https://quarto.org/docs/reference/formats/html.html#rendering>).

| Option | Description | 
|:--|:--|
| embed-resources | Produce a standalone HTML file with no external dependencies, using data: URIs to incorporate the contents of linked scripts, stylesheets, images, and videos. The resulting file should be “self-contained,” in the sense that it needs no external files and no net access to be displayed properly by a browser. This option works only with HTML output formats, including html4, html5, html+lhs, html5+lhs, s5, slidy, slideous, dzslides, and revealjs. Scripts, images, and stylesheets at absolute URLs will be downloaded; those at relative URLs will be sought relative to the working directory (if the first source file is local) or relative to the base URL (if the first source file is remote). Elements with the attribute data-external="1" will be left alone; the documents they link to will not be incorporated in the document. Limitation: resources that are loaded dynamically through JavaScript cannot be incorporated; as a result, some advanced features (e.g. zoom or speaker notes) may not work in an offline “self-contained” reveal.js slide show. |
| self-contained  | Produce a standalone HTML file with no external dependencies. Note that this option has been deprecated in favor of embed-resources. |
